### PR TITLE
Feature/add loading spinner

### LINKED ISSUE #59: Loading Widget - Make it obvious when loading is taking place so users cannot submit twice
### Problem:
Currently there is loading going on in the FE, and it may cause users to submit twice, or drop off, etc... add loading.

### DoD:
- [ ] Add a loading widget that keeps users from submitting twice, or getting into bad states
- [ ] After a submission is completed, or other loading state takes place, users should be limited from taking any action until loading ends.

### PR DESCRIPTION
**Pull request by KatunaNorbert**
Fixes #59

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "build": "rollup -c",
     "dev": "rollup -c -w",
-    "start": "sirv public"
+    "start": "sirv public --single --port 5555"
```
Comment (idiom-bytes) [inline frontend/package.json:8]: Am wondering why this change in port is required. Not too familiar with `sirv public` command.

File: frontend/src/stores/ethers.js [hunk 2]
```diff
@@ -65,7 +65,11 @@ export const connectWalletFromLocalStorage = async () => {
 
 export const signMessage = async (msg, signer) => {
   let signedMessage;
-  signedMessage = await signer.signMessage(msg);
+  try{
+    signedMessage = await signer.signMessage(msg);
+  }catch(error){
+    throw error
+  }
```
Comment (idiom-bytes) [inline frontend/src/stores/ethers.js:72]: Thanks for wrapping this in a try/catch/throw. I looked at the other functions and this looks properly implemented. I still need to do build/deploy/test locally but I was reviewing this PR and it looks good.

Closing initial review, and continuing on via local testing.

**Review by idiom-bytes — APPROVED** [review]
Looks good. I still haven't ran locally, that would be my next step.

File: frontend/package.json (hunk 1 above)
Comment (KatunaNorbert) [reply frontend/package.json:8]: It wasn't working for me with the default port(only the home page) when running only the frontend, so adding those attributes fixed it

**Comment by trizin** [comment]
Looks amazing

https://user-images.githubusercontent.com/25263018/161629594-de3a2f93-6322-487a-bdb8-197ba64e89d6.mp4

!
